scripts/apps/ingest/directives/IngestSourcesContent.ts

```diff
@@ -580,6 +581,14 @@ export function IngestSourcesContent(ingestSources, notify, api, $location,
                     return feedingService ? feedingService.templateUrl : '';
                 };
 
+                /**
+                 * Do URL request specified in url_request field
+                 * @param {Object} field
```

Comment:
Use TypeScript to specify the types of parameters. It would also be good to specify an exact type, not only that it's an object.

Reply:
alright, I was just doing the same way as for other functions in this file, but it's definitely better to use typescript, I'll update it.